torch/_dynamo/logging.py

```diff
@@ -52,6 +52,6 @@ def get_step_logger(logger):
     step = next(_step_counter)
 
     def log(level, msg):
-        logger.log(level, f"Step {step}: {msg}")
+        logger.log(level, "Step %s: %s", step, msg)
```

Comment:
This actually looks like a false positive bug in ruff, but the change is for the better anyway. It should be ignoring fstrings since we have a different (disabled) rule for it). This should probably filed as an issue that there are false positives when the first arg to logger is logging.level